decidim-proposals/lib/decidim/proposals/component.rb

```diff
@@ -55,6 +55,9 @@
     settings.attribute :comments_blocked, type: :boolean, default: false
     settings.attribute :creation_enabled, type: :boolean
     settings.attribute :proposal_answering_enabled, type: :boolean, default: true
+    settings.attribute :amendment_creation_enabled, type: :boolean, default: true
+    settings.attribute :amendment_reaction_enabled, type: :boolean, default: true
```

Comment:
Do we need some migration for the existing components?

Reply:
Let me get back to you in a sec.

Reply:
I did the following test with success to make sure as recommended by @agustibr 

1. Generate a new `development_app` in `master` branch.
2. Set up two processes:
- With `amendments_enabled`: **true**
- With `amendments_enabled`: **false**
3. Changed to `feat/amendable_by_step_config` branch and restart the application.
4. Check if the two processes had the same backend configuration and frontend behaviour.

The default value of the amendments step settings is true, and they all depend on the global setting so everything works as before.